test_scripts/Polices/appID_Management/TS01.lua

```diff
@@ -0,0 +1,83 @@
+---------------------------------------------------------------------------------------------
```

Comment:
@dboltovskyi Please rename the script using rules described in https://adc.luxoft.com/confluence/display/APPLINK/Rule%3A+Work+with+Test+Case